public/scripts/main.js

```diff
@@ -22,18 +22,18 @@ require.config({
   }
 });
 
-define(["bramble-editor", "sso-override"], function(BrambleEditor) {
+define(["bramble-editor"], function(BrambleEditor) {
   var thimbleScript = document.getElementById("thimble-script");
   var appUrl = thimbleScript.getAttribute("data-app-url");
-  var makeDetails = thimbleScript.getAttribute("data-make-details");
+  var project = thimbleScript.getAttribute("data-make-details");
```

Comment:
since we are changing it completely, let's change the attribute to `data-project-metadata` too